### ISSUE TITLE
Add support for flow_control_pin

### ISSUE THREAD

**Issue by SebuZet** [lanwin/esphome_samsung_ac#18]
Hi,

First of all thanks for great work. I was wondering if it is possible to use modbus device instead of uart?
I've got device with MAX485 chip and it won't work correctly without flow control. Maybe you can add a flow control pin supprt?

More info: https://electronics.stackexchange.com/questions/153500/correctly-using-re-and-de-with-rs485

Thanks,
Seba

**Comment by lanwin** [comment]
Hi Seba, what do you mean by flow_control_pin? For UART?

Adding modbus would make no sens. RS485 is only a transport bus, like Ethernet. This projects adds the software protocol which Samsung uses on RS485.

Modbus has both. An hardware bus and an software protocol. And both is already supported by home assist. All you need to do is to review the documentation and configure the matching registers within home assist.

**Comment by SebuZet** [comment]
Thanks for your response. I'm sorry for my ignorance. I thought Samsung HVAC is using modbus for communication between devices.
I've got [Olimex RS485](https://www.olimex.com/Products/Modules/Interface/MOD-RS485/open-source-hardware). To make it working with modbus device I had to configure **flow_control_pin** in modbus controller in ESPHome. My understanding is that MAX485 chip is a half-duplex chip. It has to be switched between read and write modes using this pin.
After connecting this to my AC I've got connection error (E01). I thought it is because of that.
